scene_execute_command.lua

```diff
@@ -88,14 +89,30 @@ function handle_scene_change()
 	local scene = obs.obs_frontend_get_current_scene()
 	local scene_name = obs.obs_source_get_name(scene)
 	local scene_enabled = obs.obs_data_get_bool(settings, "scene_enabled_" .. scene_name)
+	local capture_script_output = obs.obs_data_get_bool(settings, "capture_script_output")
+
 	if scene_enabled then
 		local command = obs.obs_data_get_string(settings, "command")
 		local scene_value = obs.obs_data_get_string(settings, "scene_value_" .. scene_name)
 		local scene_command = string.gsub(command, "SCENE_VALUE", scene_value)
 		obs.script_log(obs.LOG_INFO, "Activating " .. scene_name .. ". Executing command:\n  " .. scene_command)
-		os.execute(scene_command)
+
+		if capture_script_output then
```

Comment:
I would move the whole if ... else block into the function `capture_output` and rename that function to `execute_command`